src/ess/livedata/dashboard/widgets/plot_grid_tab.py

```diff
@@ -0,0 +1,99 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2025 Scipp contributors (https://github.com/scipp)
+from __future__ import annotations
+
+import holoviews as hv
+import panel as pn
+
+from ess.livedata.dashboard.job_controller import JobController
+from ess.livedata.dashboard.job_service import JobService
+from ess.livedata.dashboard.plotting_controller import PlottingController
+
+from .job_plotter_selection_modal import JobPlotterSelectionModal
+from .plot_grid import PlotGrid
+
+
+class PlotGridTab:
+    """Tab widget that orchestrates PlotGrid with modal workflow for plot creation."""
+
+    def __init__(
+        self,
+        *,
+        job_service: JobService,
+        job_controller: JobController,
+        plotting_controller: PlottingController,
+    ) -> None:
+        """
+        Initialize PlotGridTab.
+
+        Parameters
+        ----------
+        job_service:
+            Service for accessing job data
+        job_controller:
+            Controller for job operations
+        plotting_controller:
+            Controller for creating plotters
+        """
+        self._job_service = job_service
+        self._job_controller = job_controller
+        self._plotting_controller = plotting_controller
+
+        # Create PlotGrid (3x3 fixed)
+        self._plot_grid = PlotGrid(
+            nrows=3, ncols=3, plot_request_callback=self._on_plot_requested
+        )
```

Comment:
Medium term we will probably add a way to add new tabs with a custom row and column count.